File: Modul 1/main.py
# Modul 1
# Kegiatan 1
numbers = []

length = int(input('Input panjang list : '))
print(f'\nPanjang list : {length}\n')

for index in range(0, length):
    element = int(input(f'Input element ke {index + 1} : '))
    numbers.append(element)

print(f'\nIsi List {numbers}')

#Robb

print('\n<<<Kegiatan 1>>>\n')

for number in numbers:  
    if number > 1:
        isPrime = True
        for i in range(2, number):
            if number % i == 0:
                isPrime = False
        if isPrime:
            print(f'{number} termasuk bilangan Prima')
        elif number % 2 == 0:
            print(f'{number} termasuk bilangan Genap')
        else:
            print(f'{number} termasuk bilangan Ganjil')
    elif number == 1:
        print(f'{number} termasuk bilangan Ganjil')

#Kegiatan 2

print('\n<<<Kegiatan 2>>>\n')

def isPrimeNumber(number):
    for i in range(2, number):
        if number % i == 0:
            return False
    return True

# def isEvenNumber(number):
#     return number % 2 == 0

isEvenNumber = lambda x: x % 2 == 0

for number in numbers:
    if number > 1:
        if isPrimeNumber(number):
            print(f'{number} termasuk bilangan Prima')
        elif isEvenNumber(number):
            print(f'{number} termasuk bilangan Genap')
        else:
            print(f'{number} termasuk bilangan Ganjil')
    elif number == 1:
        print(f'{number} termasuk bilangan Ganjil')

##Robb

File: Modul 1/main.js
numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

console.log('\n<<Kegiatan 1>>\n')

for (number of numbers) {
    if (number == 1) {
        console.log(`${number} termasuk bilangan Ganjil`)
    } else {
        let isPrime = true
        for (i = 2; i < number; i++) {
            if (number % i == 0) {
                isPrime = false
            }
        }
        if (isPrime) {
            console.log(`${number} termasuk bilangan Prima`)
        } else if (number % 2 == 0) {
            console.log(`${number} termasuk bilangan Genap`)
        } else {
            console.log(`${number} termasuk bilangan Ganjil`)
        }
    }
}

console.log('\n<<Kegiatan 2>>\n')

function isPrimeNumber (number) {
    for (i = 2; i < number; i++) {
        if (number % i == 0) return
    }
    return true
}

const isEvenNumber = (number) => number % 2 == 0

numbers.map(function(number) {
    if (number == 1) {
        console.log(`${number} termasuk bilangan Ganjil`)
    } else {
        if (isPrimeNumber(number)) {
            console.log(`${number} termasuk bilangan Prima`)
        } else if (isEvenNumber(number)) {
            console.log(`${number} termasuk bilangan Genap`)
        } else {
            console.log(`${number} termasuk bilangan Ganjil`)
        }
    }
})
